src/registrar/models/domain.py

```diff
@@ -1100,14 +1102,28 @@ def is_expired(self):
         now = timezone.now().date()
         return self.expiration_date < now
 
-    def state_display(self):
+    def is_expiring(self):
+        """
+        Check if the domain's expiration date is within 60 days.
+        Return False bc there's no expiration date meaning so not expiring
```

Comment:
nit: Edit comment to more readable -- eg Return True if domain expiration exists and within 60 days

Reply:
Just pushed the update!

Reply:
thank you! I would also remove the return false line since the info is repeated in the true statement. If you want you can add " and False otherwise" to line 1109 if you feel strongly about making the distinction

Reply:
thank you! I would also remove the return false line since the info is repeated in the true statement. If you want you can add " and False otherwise" to line 1109 if you feel strongly about making the distinction

Reply:
Updated!